src/JackTrip.cpp

```diff
@@ -1156,7 +1158,10 @@ void JackTrip::tcpTimerTick()
             serverHostAddress = info.addresses().constFirst();
         }
     }
-    mTcpClient.connectToHost(serverHostAddress, mTcpServerPort);
+
+    if (mTcpClient.state() == QAbstractSocket::UnconnectedState) {
```

Comment:
There were some errors I found during this work when restarting jacktrip quickly where the `connectToHost()` call failed because the connection was in-progress or already connected. So I added this check here to only `connectToHost()` when it's unconnected.